### PR TITLE
doc: Asset Tracker v2: added Selecting build type section

### PR DESCRIPTION
**Pull request by peknis**
Added instructions on selecting the build type.

Ref: https://projecttools.nordicsemi.no/jira/browse/NCSDK-14430

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Review by greg-fer — CHANGES_REQUESTED** [review]
I welcome the change, but I have some doubts:

- "Selecting a build type" always comes with a section about application build types in the overview, which is missing here.
- When checking the application source folders, I see a lot of overlays, but only one prj.conf with no prj_release.conf (which is mentioned in the added section).
- Further checking the source files, I don't see any other build type prj.conf files, except for the lonely prj_quemu_x86.conf and boards/nrf9160dk_nrf9160_ns.conf.

In other words, we need a section that describes the available build types and more customized examples in the Selecting section.

**Comment by peknis** [comment]
> I welcome the change, but I have some doubts:
> 
> * "Selecting a build type" always comes with a section about application build types in the overview, which is missing here.
> * When checking the application source folders, I see a lot of overlays, but only one prj.conf with no prj_release.conf (which is mentioned in the added section).
> * Further checking the source files, I don't see any other build type prj.conf files, except for the lonely prj_quemu_x86.conf and boards/nrf9160dk_nrf9160_ns.conf.
> 
> In other words, we need a section that describes the available build types and more customized examples in the Selecting section.

I agree, I used the nRF Desktop application as the model for this addition and I noticed it listed separately the build types.
@simensrostad, any chance to reach your helping hand here? To write a bit of the application-specific build types in the overview, to start with.

**Comment by simensrostad** [comment]
As @greg-fer pointed out we only have `prj.conf` and `prj_qemu_x86.conf` which should be used when building for thingy and DK (former) and QEMU x86 (latter). This happens automatically when building with west if not specified differently with the`-DCONF_FILE` argument. But, in VS code and Segger IDE you have to actually select the config.

This differs a bit from how it is done in nRF Desktop where there seem to be multiple `prj_x.conf` files that can be used per target board. So I'm not completely sure of this section applies for the ATv2.

In section [Configuration files](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7186/nrf/applications/asset_tracker_v2/doc/asset_tracker_v2_description.html#configuration-files) the different configuration files are already listed with some explanation.

**Comment by peknis** [comment]
Build types are not relevant for ATv2 at the moment. Closing the PR.
